torchao/quantization/quant_api.py

```diff
@@ -287,3 +297,103 @@ def filter_fn(module, fqn):
         _is_linear if filter_fn is None else filter_fn,
     )
     return model
+
+def get_apply_8da4w_quant(groupsize=32):
```

Comment:
should we make these private for now? I'm not sure if we want users to call these directly

Reply:
it's fine I think, I'm planning to use this in an example as well. what's the concern of this being public?